packages/functional-tests/tests/misc/authClientV2.spec.ts

```diff
@@ -2,29 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { test, expect } from '../../lib/fixtures/standard';
+import { test, expect, PASSWORD } from '../../lib/fixtures/standard';
 import AuthClient, {
   getCredentialsV2,
   getCredentials,
 } from 'fxa-auth-client/browser';
-import crypto from 'crypto';
-import { SaltVersion } from 'fxa-auth-client/lib/salt';
-
-let curClient: AuthClient | undefined;
-function getClient(url: string, version: SaltVersion) {
-  curClient = new AuthClient(url, {
-    keyStretchVersion: version,
-  });
-  return curClient;
-}
 
 test.describe('auth-client-tests', () => {
-  let email = '';
-  let password = '';
-  let credentials: any;
-
-  async function signUp(client: AuthClient) {
-    credentials = await client.signUp(email, password, {
+  async function signUp(client: AuthClient, email: string, password: string) {
```

Comment:
It's fine to remove pass these as params, but scoped variables seem okay too. What's the benefit of doing it this way?